vm_supervisor/network/interfaces.py

```diff
@@ -27,18 +28,19 @@ def host_ip(self) -> IPv4Interface:
     async def create(self):
         logger.debug("Create network interface")
 
-        run(["/usr/bin/ip", "tuntap", "add", self.device_name, "mode", "tap"])
+        ip_command = whereis_binary("ip")
```

Comment:
Why not use the builtin `shutil.which` ? https://docs.python.org/3.9/library/shutil.html#shutil.which

Reply:
done